data/net/minecraft/world/level/levelgen/WorldgenRandom.mapping

```diff
@@ -1,7 +1,18 @@
 CLASS net/minecraft/world/level/levelgen/WorldgenRandom
+	METHOD <init> (J)V
+		ARG 1 seed
+	METHOD seedSlimeChunk (IIJJ)Ljava/util/Random;
+		ARG 0 chunkX
+		ARG 1 chunkZ
+		ARG 2 seed
 	METHOD setBaseChunkSeed (II)J
 		ARG 1 x
 		ARG 2 z
+	METHOD setBaseStoneSeed (JIII)J
+		ARG 1 seed
+		ARG 3 x
+		ARG 4 y
+		ARG 5 z
 	METHOD setDecorationSeed (JII)J
```

Comment:
This is not a comment on the changes in this PR as much, but I'm pretty sure most of these `x` / `z` mappings in the `setSeed` method are wrong. As they are not coordinates, rather just different seeds. e.g. the ones used for biome feature placement are the generation step ordinal, and the cumulative feature index. Might be worth revisiting, and/or looking for callsites. (or, cross referencing w/ [yarn's mappings](https://github.com/FabricMC/yarn/blob/1.17.1-pre3/mappings/net/minecraft/world/gen/ChunkRandom.mapping) as they are fairly comprehensive here)

Reply:
I looked at usage of `WorldgenRandom#setBaseChunkSeed` and `WorldgenRandom#setDecorationSeed`. Both were used with ChunkPos x and z. Because of this, I'd say x/z is correct. If I understood correctly.

Reply:
Fair enough, I'd have to double check later myself, but will probably leave that for Project Phoenix.